client/src/components/Map/map.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// eslint-disable-next-line
+import 'mapbox-gl/dist/mapbox-gl.css';
+
+import mapboxgl from 'mapbox-gl/dist/mapbox-gl';
+
+//eslint-disable-next-line
+import MapboxWorker from 'worker-loader!mapbox-gl/dist/mapbox-gl-csp-worker';
+import Mapbox from 'react-map-gl';
+import PropTypes from 'prop-types';
+
+mapboxgl.workerClass = MapboxWorker;
+
+
+export const Map = ({ children, onClick, initialViewState, interactiveLayerIds, style }) => {
+	return (
+		<Mapbox
+			mapboxAccessToken="pk.eyJ1IjoieW9zc2ktZXluYXYiLCJhIjoiY2xjcDgwcDJpMXJldTNybXFzaXNwd2FpYSJ9.JQLLuh4VYjrxgpxWIrYsGA"
```

Comment:
should be extracted outside (no security issue since it will be used in client side anyway)

Reply:
we have client side config, can be added there